package.json

```diff
@@ -10,12 +10,15 @@
     "build": "next build",
     "start": "next start",
     "lint": "eslint --cache --fix ./src",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "prettier": "prettier --write src",
```

Comment:
You don't really need this command, just run `lint` - it will run prettier behind the scene and fix everything